src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -422,6 +424,7 @@ export default function AuthLayoutRouter (props) {
           </div>
           <SocketListener location={location} groupSlug={currentGroupSlug} />
           <SocketSubscriber type='group' id={get('slug', currentGroup)} />
+          {/* <SocketSubscriber type='user' id={currentUser.id} /> */}
```

Comment:
This still relevant?

Reply:
could be someday, if we want to update user stuff in real time. but i can remove for now